docs/features.md

```diff
@@ -0,0 +1,68 @@
+# Ngeo Features
+
+This document lists ngeo's features and if they are visible/configurable in the mobile interface.
+
+Feature | Sub-feature | Desktop | Mobile
+------- | ----------- | ------- | ------
+Navigation | Zoom&pan | X | X
+. | Map rotation | (2.3) | (2.3)
+. | Geolocation | X | X
+. | Navigation bar | X | -
+Full-text search | Data | X | X
+. | Layers/groups/themes | X | -
+. | Coordinates | X | X
+Data | Background selector | X | X
+. | Themes (flush) | X | X
+. | Themes (non-flush) | X | (2.3)
+Layers | (Un)select layer/group | X | X
+. | Clear all groups | X | X
+. | Remove group | X | X
+. | Reorder groups | X | -
+. | Legend | X | X
+. | Go to scale | X | X
+. | Info on layer | X | X
+. | Opacity | X | (2.3)
+. | Time layers | X | X
+Query | Point query | X | X
+. | Rectangular query | X | X
+. | Results window | X | X
+. | Results window filter | X | -
+. | Results grid | X | X
+. | Results grid zoom to | X | -
+. | Results grid export | X | -
+. | Infos on point (includes raster) | X | -
+Access rights | Login | X | X
+. | Lost password | X | X
+. | Rights on layers/themes/... | X | X
+Print | PDF | X | -
+. | PNG | X | -
+Drawing/measure | Drawing | X | -
+. | Measure point | X | X
+. | Measure line | X | X
+. | Measure polygon | X | -
+. | Measure circle/azimuth | X | -
+. | Measure rectangle | X | -
+Filters | Spatial filter | X | -
+. | Attributes filter | X | -
+. | WMS filter | X | -
+. | WFS filter | X | -
+. | Directed filters | X | -
+. | Advanced filters | X | -
+. | Saved filters | X | -
+Editing | Edit point | X | -
+. | Edit line | X | -
+. | Edit polygon | X | -
+. | Snapping | X | -
+Profile | Draw profile | X | -
+. | CSV export | X | -
+StreetView | StreetView panel | X | -
+External layers | Online WMS | (2.3) | -
+. | Online WMTS | (2.3) | -
+. | Local KML | (2.3) | -
+. | Local GPX | (2.3) | -
+. | Query point on WMS | (2.3) | -
+Permalink | Share window | X | -
+. | Location | X | X
+. | Layers/groups/themes | X | X
+. | Drawing | X | (2.4?)
```

Comment:
mobile 2.4 read only

Reply:
Also to add: Object Editing features (viewer+editor). See with ELU